setup.cfg

```diff
@@ -27,8 +27,8 @@ include_package_data = True
 python_requires = >=3.8
 install_requires = 
     attr
-    jsons
     iso3166
+    marshmallow
```

Comment:
und hier braucht's das marshmallow-enum und stringcase nicht?

Reply:
ah doch, die setup.cfg vergesse ich oft, da tox mir die dependencies installiert.
muss mal schauen wie man dafür einen test aufsetzt.
Man müsste bo4e alleine installieren und pip selbst die dependencies installieren lassen und danach dann die eigentlich tests wieder laufen lassen.

Reply:
Habe nun `marshmallow-enum` und `stringcase` hinzugefügt.
[➕ Add marshmallow and marshmallow-enum to dependencies](https://github.com/Hochfrequenz/BO4E-python/pull/27/commits/f6e3167201a14076e5189ff36e00c56bb8e95d73)